plugin/pkg/scheduler/factory/factory.go

```diff
@@ -123,6 +133,106 @@ func NewConfigFactory(client *client.Client, schedulerName string) *ConfigFactor
 						c.modeler.ForgetPodByKey(t.Key)
 					}
 				})
+
+				pod := obj.(*api.Pod)
+				if pod.Spec.NodeName != "" {
+					c.EquivalencePodCache.SendInvalidNodeCacheReq(pod.Spec.NodeName)
+				}
+			},
+		},
+	)
+
+	c.NodeLister.Store, c.nodePopulator = framework.NewInformer(
```

Comment:
@wojtek-t I am refactoring this, but I'm not sure which part are you referring is unneeded here?